apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss

```diff
@@ -0,0 +1,264 @@
+			168px -100.66667px $color5,
+			-244px 24.33333px $color6,
+			97px -325.66667px $color5,
+			-211px -182.66667px $color3,
+			236px -126.66667px $color4,
+			140px -196.66667px $color1,
+			125px -175.66667px $color2,
+			118px -381.66667px $color5,
+			144px -111.66667px $color6,
+			36px -78.66667px $color4,
+			-63px -196.66667px $color2,
+			-218px -227.66667px $color3,
+			-134px -377.66667px $color4,
+			-36px -412.66667px $color4,
+			209px -106.66667px $color3,
+			91px -278.66667px $color1,
+			-22px -191.66667px $color3,
+			139px -392.66667px $color3,
+			56px -2.66667px $color1,
+			-156px -276.66667px $color5,
+			-163px -233.66667px $color3,
+			-238px -346.66667px $color3,
+			62px -363.66667px $color1,
+			244px -170.66667px $color1,
+			224px -142.66667px $color2,
+			141px -208.66667px $color2,
+			211px -285.66667px $color5,
+			181px -128.66667px $color1,
+			90px -123.66667px $color2,
+			189px 70.33333px $color3,
+			-18px -383.66667px $color3,
+			100px -6.66667px $color5;
+	}
+
+	> .event-countdown__fireworks-after {
+		animation-delay: 1.25s, 1.25s, 1.25s;
+		animation-duration: 1.25s, 1.25s, 6.25s;
+	}
+}
+
+
+@-webkit-keyframes bang {
+	from {
+		box-shadow: 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white;
+	}
+}
+
+@keyframes bang {
+	from {
+		box-shadow: 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white;
+	}
+}
+
+@-webkit-keyframes gravity {
+	to {
+		transform: translateY( 200px );
+		opacity: 0;
+	}
+}
+
+@keyframes gravity {
+	to {
+		transform: translateY( 200px );
+		-webkit-transform: translateY( 200px );
+		opacity: 0;
+	}
+}
+
+@-webkit-keyframes position {
+	0%, 19.9% {
+		margin-top: 10%;
+		margin-left: 40%;
+	}
+
+	20%, 39.9% {
+		margin-top: 40%;
+		margin-left: 30%;
+	}
+
+	40%, 59.9% {
+		margin-top: 20%;
+		margin-left: 70%;
+	}
+
+	60%, 79.9% {
+		margin-top: 30%;
+		margin-left: 20%;
+	}
+
+	80%, 99.9% {
+		margin-top: 30%;
+		margin-left: 80%;
+	}
+}
+
+	@keyframes position {
```

Comment:
Keyframes end up in global CSS scope, hence colliding with anything with same name and this is pretty generic. I'd suggest prefixing these with `jetpack_`, `event_countdown_`, or similar.

Reply:
Thanks, I addressed in this PR: https://github.com/Automattic/wp-calypso/pull/40926